L1Trigger/TrackTrigger/src/L1TrackQuality.cc

```diff
@@ -156,6 +166,16 @@ void L1TrackQuality::setL1TrackQuality(TTTrack<Ref_Phase2TrackerDigi_>& aTrack)
   }
 }
 
+float L1TrackQuality::runEmulatedTQ(std::vector<ap_fixed<10,5>> inputFeatures) {
+    // load in bdt
+    
+    conifer::BDT<ap_fixed<10,5>,ap_fixed<10,5>> bdt(this->ONNXmodel_.fullPath());
+
+    // collect features and classify using bdt
+    std::vector<ap_fixed<10,5>> output = bdt.decision_function(inputFeatures);
+    return output.at(0).to_float(); // need logistic sigmoid fcn applied to xgb output
```

Comment:
Is this missing the sigmoid function? It doesn't seem like it's actually being implemented

Reply:
The FW doesn't implement a sigmoid function so the emulation doesn't either. This means the binning of the track quality output is also without a sigmoid function